misc/ls-extensions/modules/performance-analyzer-services/src/main/java/io/ballerina/PerformanceAnalyzerNodeVisitor.java

```diff
@@ -307,6 +314,42 @@ public void visit(AssignmentStatementNode assignmentStatementNode) {
         assignmentStatementNode.expression().accept(this);
     }
 
+    @Override
+    public void visit(NamedWorkerDeclarationNode namedWorkerDeclarationNode) {
+
+        if (withinRange) {
+            withinWorker = true;
+            if (!isWorkerExists) {
+                isWorkerExists = namedWorkerDeclarationNode != null;
```

Comment:
Can't we assume namedWorkerDeclarationNode is not null since its visit is invoked?

Reply:
Can't access the worker body without a null check. 
`Method invocation 'workerBody' may produce 'NullPointerExceptio`